src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/ProfitPerExcavation.kt

```diff
@@ -0,0 +1,46 @@
+package at.hannibal2.skyhanni.features.mining.fossilexcavator
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.mining.FossilExcavationEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.CollectionUtils.sortedDesc
+import at.hannibal2.skyhanni.utils.ItemUtils.itemName
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.NEUItems.getPrice
+import at.hannibal2.skyhanni.utils.NumberUtil
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class ProfitPerExcavation {
+    private val config get() = SkyHanniMod.feature.mining.fossilExcavator
+
+    @SubscribeEvent
+    fun onFossilExcavation(event: FossilExcavationEvent) {
+        if (!config.profitPerExcavation) return
+        val loot = event.loot
+
+        var totalProfit = 0.0
+        val map = mutableMapOf<String, Double>()
+        loot.forEach { (name, amount) ->
+            NEUInternalName.fromItemNameOrNull(name)?.let {
+                val profit = amount * it.getPrice()
+                val text = "Found $name §8${amount.addSeparators()}x §7(§6${NumberUtil.format(profit)}§7)"
+                map[text] = profit
+                totalProfit += profit
+            }
+        }
+
+        val scrapItem = FossilExcavatorAPI.scrapItem
+
+        val scrapPrice = scrapItem.getPrice()
```

Comment:
```suggestion
        val scrapPrice = FossilExcavatorAPI.scrapItem.getPrice()
```

imo a bit cleaner

Reply:
We use it twice
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/0a7f8a5f-b6ff-4ecf-bbad-b32024ba5384)

not sure if this is truly cleaner

Reply:
Oh didn't see this
nvm then